examples/DC2_examples/DC2_gt_profiles.ipynb

```diff
@@ -53,7 +53,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "object_cat = GCRCatalogs.load_catalog('dc2_object_run2.1i_dr1')"
+    "object_cat = GCRCatalogs.load_catalog('dc2_object_run2.2i_dr3')"
```

Comment:
Actually, let's make the default `dc2_object_run2.2i_dr6`. I corresponds to 5 years of LSST data (while dr3 is 2 years) and there doesn't seem to be any specific warning with that catalog.